2025-05-20 20:53:10.660 [info] Extension host with pid 17068 started
2025-05-20 20:53:10.660 [info] Skipping acquiring lock for /home/dimgerasimou/.config/VSCodium/User/workspaceStorage/1882ba2a21686c8358f9e3d8dcde57e1.
2025-05-20 20:53:10.882 [info] ExtensionService#_doActivateExtension vscode.emmet, startup: false, activationEvent: 'onLanguage'
2025-05-20 20:53:10.896 [info] ExtensionService#_doActivateExtension luma.jupyter, startup: false, activationEvent: 'onLanguage:python'
2025-05-20 20:53:11.190 [info] ExtensionService#_doActivateExtension ms-python.python, startup: false, activationEvent: 'onLanguage:python', root cause: ms-python.debugpy
2025-05-20 20:53:11.316 [info] ExtensionService#_doActivateExtension ms-toolsai.jupyter, startup: false, activationEvent: 'onLanguage:python'
2025-05-20 20:53:11.511 [info] ExtensionService#_doActivateExtension vscode.configuration-editing, startup: false, activationEvent: 'onLanguage:jsonc'
2025-05-20 20:53:11.516 [info] ExtensionService#_doActivateExtension vscode.json-language-features, startup: false, activationEvent: 'onLanguage:jsonc'
2025-05-20 20:53:11.554 [info] ExtensionService#_doActivateExtension vscode.typescript-language-features, startup: false, activationEvent: 'onLanguage:jsonc'
2025-05-20 20:53:11.593 [info] ExtensionService#_doActivateExtension ms-toolsai.jupyter-renderers, startup: false, activationEvent: 'api', root cause: ms-toolsai.jupyter
2025-05-20 20:53:12.319 [info] ExtensionService#_doActivateExtension vscode.git-base, startup: true, activationEvent: '*', root cause: vscode.git
2025-05-20 20:53:12.321 [info] ExtensionService#_doActivateExtension mhutchie.git-graph, startup: true, activationEvent: '*'
2025-05-20 20:53:12.370 [info] ExtensionService#_doActivateExtension ms-python.debugpy, startup: false, activationEvent: 'onLanguage:python'
2025-05-20 20:53:12.427 [info] ExtensionService#_doActivateExtension vscode.git, startup: true, activationEvent: '*'
2025-05-20 20:53:12.461 [info] ExtensionService#_doActivateExtension vscode.github, startup: true, activationEvent: '*'
2025-05-20 20:53:12.556 [info] Eager extensions activated
2025-05-20 20:53:12.578 [info] ExtensionService#_doActivateExtension vscode.debug-auto-launch, startup: false, activationEvent: 'onStartupFinished'
2025-05-20 20:53:12.581 [info] ExtensionService#_doActivateExtension vscode.merge-conflict, startup: false, activationEvent: 'onStartupFinished'
2025-05-20 20:53:12.617 [info] ExtensionService#_doActivateExtension Catppuccin.catppuccin-vsc, startup: false, activationEvent: 'onStartupFinished'
2025-05-20 20:53:12.625 [info] ExtensionService#_doActivateExtension Catppuccin.catppuccin-vsc-icons, startup: false, activationEvent: 'onStartupFinished'
2025-05-20 20:53:12.695 [info] ExtensionService#_doActivateExtension vadimcn.vscode-lldb, startup: false, activationEvent: 'onStartupFinished'
2025-05-20 20:53:14.532 [info] ExtensionService#_doActivateExtension vscode.npm, startup: false, activationEvent: 'onTerminalQuickFixRequest:ms-vscode.npm-command'
2025-05-20 22:22:28.571 [info] ExtensionService#_doActivateExtension vscode.ipynb, startup: false, activationEvent: 'onNotebookSerializer:jupyter-notebook'
2025-05-20 22:22:28.585 [info] ExtensionService#_doActivateExtension vscode.extension-editing, startup: false, activationEvent: 'onLanguage:json'
2025-05-20 22:26:38.669 [info] ExtensionService#_doActivateExtension ms-toolsai.vscode-jupyter-cell-tags, startup: false, activationEvent: 'onNotebook:jupyter-notebook'
2025-05-20 22:26:38.673 [info] ExtensionService#_doActivateExtension ms-toolsai.vscode-jupyter-slideshow, startup: false, activationEvent: 'onNotebook:jupyter-notebook'
2025-05-20 22:27:01.986 [info] ExtensionService#_doActivateExtension vscode.markdown-language-features, startup: false, activationEvent: 'onLanguage:markdown'
2025-05-20 22:27:02.343 [info] ExtensionService#_doActivateExtension vscode.markdown-math, startup: false, activationEvent: 'api', root cause: vscode.markdown-language-features
2025-05-20 22:34:46.764 [error] [ms-python.python] provider FAILED
2025-05-20 22:34:46.764 [error] Error: UnboundLocalError: cannot access local variable 'start_index' where it is not associated with a value
    at ae (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1965471)
    at oe (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1963431)
    at Immediate.<anonymous> (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1959725)
    at processImmediate (node:internal/timers:483:21)
2025-05-20 22:42:03.074 [error] [ms-python.python] provider FAILED
2025-05-20 22:42:03.074 [error] Error: UnboundLocalError: cannot access local variable 'start_index' where it is not associated with a value
    at ae (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1965471)
    at oe (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1963431)
    at Immediate.<anonymous> (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1959725)
    at processImmediate (node:internal/timers:483:21)
2025-05-20 22:42:04.143 [error] [ms-python.python] provider FAILED
2025-05-20 22:42:04.143 [error] Error: UnboundLocalError: cannot access local variable 'start_index' where it is not associated with a value
    at ae (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1965471)
    at oe (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1963431)
    at Immediate.<anonymous> (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1959725)
    at processImmediate (node:internal/timers:483:21)
2025-05-20 23:03:49.110 [error] [ms-python.python] provider FAILED
2025-05-20 23:03:49.111 [error] Error: UnboundLocalError: cannot access local variable 'start_index' where it is not associated with a value
    at ae (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1965471)
    at oe (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1963431)
    at Immediate.<anonymous> (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1959725)
    at processImmediate (node:internal/timers:483:21)
2025-05-20 23:24:33.174 [error] [ms-python.python] provider FAILED
2025-05-20 23:24:33.174 [error] Error: UnboundLocalError: cannot access local variable 'end_index' where it is not associated with a value
    at ae (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1965471)
    at oe (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1963431)
    at Immediate.<anonymous> (/home/dimgerasimou/.vscode-oss/extensions/ms-python.python-2025.4.0-universal/out/client/extension.js:2:1959725)
    at processImmediate (node:internal/timers:483:21)
2025-05-21 01:37:26.640 [info] Extension host terminating: renderer closed the MessagePort
2025-05-21 01:37:26.679 [info] Extension host with pid 17068 exiting with code 0
